plugins/lighthouse/src/components/AuditList/index.tsx

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
my $.02: I find that it can make auto-import a little annoying. It ends up guessing between which of the files you mean if you have a lot of passthrough importing going on. I like having a clean `index.ts` at the top of a package to compose an API in a very clear manner to the consumer (plus you can unit test that all of the exports you document are present, that sort of thing), but throughout the package itself I don't see the point.